### PR TITLE
Update utils packages and documentation

### PR DESCRIPTION
**Pull request by vivianjeng**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally in root directory and any changes were pushed
- [x] Lint (`yarn lint:fix`) was run locally in root directory and any changes were pushed

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/docs/docs/utils-api/incremental-tree.md [hunk 1]
```diff
@@ -16,23 +20,31 @@ Get a new tree instance.
...
   arity: number = 2
 ): IncrementalMerkleTree
 ```
 
+## Node
+
+The type of each leaf.
+
+```ts
+type Node = any
```
Comment (vimwitch) [inline packages/docs/docs/utils-api/incremental-tree.md:33]: I don't like this type, we should use `bigint` to be more clear.

File: packages/docs/docs/utils-api/hashes.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: "Hash functions"
```
Comment (vimwitch) [inline packages/docs/docs/utils-api/hashes.md:4]: What do you think of putting hashes, helpers, constants all on the same page?

File: packages/docs/docs/utils-api/hashes.md [hunk 3]
```diff
@@ -49,7 +55,7 @@ genStateTreeLeaf(
 
 ## genEpochTreeLeaf
 
-Calculate a state tree leaf for a user.
+Calculate a epoch tree leaf in an [epoch tree](../protocol/trees.md#epoch-tree).
```
Comment (CJ-Rose) [inline packages/docs/docs/utils-api/hashes.md:58]: typo here - 'an' epoch tree

File: packages/docs/docs/utils-api/incremental-tree.md (hunk 1 above)
Comment (vivianjeng) [reply packages/docs/docs/utils-api/incremental-tree.md:33]: it is defined in zk-kit
https://github.com/privacy-scaling-explorations/zk-kit/blob/f9b209d6006fd454fc464bbd7e1ad3cfab6389d0/packages/incremental-merkle-tree/src/types/index.ts#L1

File: packages/docs/sidebars.js [hunk 4]
```diff
@@ -232,45 +232,35 @@ const sidebars = {
...
                     type: 'doc',
                     label: 'constants',
                     id: 'utils-api/constants',
                 },
                 {
                     type: 'doc',
-                    label: 'hash()',
+                    label: 'hashes',
                     id: 'utils-api/hashes',
                 },
```
Comment (CJ-Rose) [inline packages/docs/sidebars.js:249]: i think you forgot to delete these? i can't build docs from this brach:

> [INFO] Starting the development server...
> [ERROR] Loading of version failed for version current
> [ERROR] Error: Invalid sidebar file at "sidebars.js".
> These sidebar document ids do not exist:
> - utils-api/constants
> - utils-api/hashes

**Pull request merged**